cli/command/manifest/rm.go

```diff
@@ -0,0 +1,47 @@
+package manifest
+
+import (
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+type rmOpts struct {
+	target string
+}
+
+func newRmManifestListCommand(dockerCli command.Cli) *cobra.Command {
+	opts := rmOpts{}
+
+	cmd := &cobra.Command{
+		Use:   "rm MANIFEST_LIST",
+		Short: "Delete a manifest list from local storage",
+		Args:  cli.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			opts.target = args[0]
+			return runRm(dockerCli, opts)
+		},
+	}
+
+	return cmd
+}
+
+func runRm(dockerCli command.Cli, opts rmOpts) error {
+
+	targetRef, err := normalizeReference(opts.target)
+	if err != nil {
+		return err
+	}
+
+	manifests, err := dockerCli.ManifestStore().GetList(targetRef)
```

Comment:
question: I wonder why we search the manifest list, couldn't we just remove it and return `Remove` error?

Reply:
sorry, naive copy-paste.

fixed 30466e28d28f6722053c5a232e99ddbae8222715

Reply:
actually in working on [supporting multiple args](https://github.com/docker/cli/pull/2449/files/46c61d85e973cc9fdd28d42db9ecebe373e9b942#r412251610) it turns out that `dockerCli.ManifestStore().GetList(targetRef)` is useful because `dockerCli.ManifestStore().Remove(targetRef)` will fail silently when `targetRef` does not exist.